jqm-all/jqm-integration-tests/src/test/java/com/enioka/jqm/tools/JqmBaseTest.java

```diff
@@ -15,23 +15,22 @@
  */
 package com.enioka.jqm.tools;
 
-import java.io.InputStream;
-import java.util.ArrayList;
-import java.util.HashMap;
-import java.util.List;
-import java.util.Map;
-import java.util.Properties;
+import java.util.*;
+
+import java.sql.ResultSet;
 
 import javax.naming.NamingException;
 import javax.naming.spi.NamingManager;
 
 import com.enioka.jqm.api.JobInstance;
 import com.enioka.jqm.api.JqmClientFactory;
 import com.enioka.jqm.api.Query;
+import com.enioka.jqm.jdbc.DatabaseUnreachableException;
 import com.enioka.jqm.jdbc.Db;
 import com.enioka.jqm.jdbc.DbConn;
 import com.enioka.jqm.test.helpers.TestHelpers;
 
+import org.apache.commons.lang.NotImplementedException;
```

Comment:
useless. (also, usually would be https://docs.oracle.com/javase/8/docs/api/java/lang/UnsupportedOperationException.html)